modules/51_internalizeDamages/KotzWenzCPreg/datainput.gms

```diff
@@ -0,0 +1,19 @@
+*** |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/51_internalizeDamages/KotzWenzItr/datainput.gms
+
+* satisfy dependencies
+$ifi not %damages% == 'KotzWenz' abort "module internalizeDamages=KotzWenzItr requires module damages=KotzWenz";
+$ifi not %cm_magicc_temperatureImpulseResponse% == 'on' abort "module internalizeDamages=DiceLikeItr requires cm_magicc_temperatureImpulseResponse=on";
+
+*init carbon tax for 1st iter
```

Comment:
Is it ok to have the SCC carbon tax in 2020 already? Or should it rather be 2025?